repos/cray.repos

```diff
@@ -2,6 +2,7 @@ https://arti.dev.cray.com/artifactory/cos-rpm-stable-local/release/cos-2.3/sle15
 
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp3/                                        cray-algol60-csm-rpms-stable-sle-15sp3            --no-gpgcheck -p 88                   cray/csm/sle-15sp3
 https://artifactory.algol60.net/artifactory/csm-rpms/hpe/stable/sle-15sp2/                                        cray-algol60-csm-rpms-stable-sle-15sp2            --no-gpgcheck -p 89                   cray/csm/sle-15sp2
+https://artifactory.algol60.net/artifactory/csm-rpms/hpe/unstable/sle-15sp3/                                      cray-algol60-csm-rpms-unstable-sle-15sp3          --no-gpgcheck -p 87                   cray/csm/sle-15sp3-unstable
```

Comment:
Unsigned RPMs will not install if I remember correctly, and we do not want to add/maintain unstable repositories.

Reply:
Building a stable artifact. Will update these PRs when ready.

Reply:
Stable RPMs building now. I've updated these PRs, but the artifacts haven't been pushed to artifactory yet. Will comment again when they have.

Reply:
Stable artifacts published. This PR is now ready to merge.